ui/client/src/components/Header.tsx

```diff
@@ -39,8 +39,10 @@ export const Header: React.FC = () => {
       return 0;
     } else if (path.startsWith(AppRoutes.Submissions)) {
       return 1;
+    } else if(path.startsWith(AppRoutes.Peers)) {
```

Comment:
super nitpicky,  but just for style consistency :) 

```suggestion
    } else if (path.startsWith(AppRoutes.Peers)) {
```